src/pages/index.js

```diff
@@ -184,8 +132,9 @@ const editProfilePopup = new PopupWithForm("#edit-modal", (card) => {
   return api
     .editUserInfo(card)
     .then((res) => {
-      console.log(res);
+      editProfilePopup.setButtonText(false);
```

Comment:
`      editProfilePopup.setButtonText(false);
`

You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case.


**Please, correct it everywhere**

Reply:
completed.